packages/vscode-extension/src/debug/commonUtils.ts

```diff
@@ -238,6 +239,25 @@ export async function getLocalTeamsAppId(): Promise<string | undefined> {
   }
 }
 
+export async function getLocalBotId(): Promise<string | undefined> {
+  try {
+    if (isConfigUnifyEnabled()) {
```

Comment:
Do we have the case that an M365 app with unify config disabled?

Reply:
At current stage, unify config is disabled by default, so M365 feature also needs to check this config.